.dockerignore

```diff
@@ -1,14 +1,50 @@
 .DS_Store
+.git
 .github
 .gitignore
-.idea
 .jekyll-cache
 .jekyll-metadata
 .sass-cache
-tests
 _releaser
 _site
 CONTRIBUTING.md
+CODE_OF_CONDUCT.md
+INSTALL.md
+README.md
+/vendor
 Dockerfile
 docker-compose.yml
-/vendor
+docker-compose-dev.yml
+
+### IDEs
+
+.idea
+.vscode/
+
+### Python project ignores
+
+*.pyc
+*.pyo
+*.mo
+*.db
+*.css.map
+*.egg-info
+*.sql.gz
+.cache
+.idea/workspace.xml
+.project
+.pydevproject
+__pycache__
+dist
+docs
```

Comment:
I think this is what was causing my error

Reply:
Oh true, because we are using them so we can't remove them from the build. Good catch! 💯

Reply:
well, that was part of it... failed again on `requirements.txt` ...

let me try the other PR like you said I should :smile: